lib/packwerk/disable_sorbet.rb

```diff
@@ -0,0 +1,38 @@
+# typed: true
+# frozen_string_literal: true
+
+require "sorbet-runtime"
+
+begin
+  T::Configuration.default_checked_level = :never
+
+  T.singleton_class.prepend(
+    Module.new do
+      def cast(value, type, checked: true)
+        value
+      end
+
+      def let(value, type, checked: true)
+        value
+      end
+
+      def must(arg)
+        arg
+      end
+
+      def absurd(value)
+        value
+      end
+
+      def bind(value, type, checked: true)
+        value
+      end
+    end
+  )
+rescue RuntimeError => error
+  if /Set the default checked level earlier./.match?(error.message)
```

Comment:
It's not clear where this string comes from. Maybe extract it to a nicely named variable?

Reply:
I'll add a comment with the error's source and it happens. Basically Sorbet throws a RuntimeError if it evaluates signatures before you try to disable typechecking. We match based on the message here because it isn't a custom error type we can account for.